pagerules.go

```diff
@@ -104,18 +104,20 @@ type PageRulesResponse struct {
 // CreatePageRule creates a new Page Rule for a zone.
 //
 // API reference: https://api.cloudflare.com/#page-rules-for-a-zone-create-a-page-rule
-func (api *API) CreatePageRule(zoneID string, rule PageRule) error {
+func (api *API) CreatePageRule(zoneID string, rule PageRule) (*PageRuleDetailResponse, error) {
 	uri := "/zones/" + zoneID + "/pagerules"
 	res, err := api.makeRequest("POST", uri, rule)
 	if err != nil {
-		return errors.Wrap(err, errMakeRequestError)
+		return nil, errors.Wrap(err, errMakeRequestError)
 	}
-	var r PageRuleDetailResponse
-	err = json.Unmarshal(res, &r)
+
+	var pageRuleResp *PageRuleDetailResponse
+	err = json.Unmarshal(res, &pageRuleResp)
```

Comment:
I thought that was weird, but I don't know Go so I just copied from [dns.go](https://github.com/cloudflare/cloudflare-go/blob/master/dns.go#L55-L56) - I'll fix it though if it is alright without. :slightly_smiling_face: